docs/templates/index.html

```diff
@@ -102,7 +103,7 @@ <h2>Augmented Markdown</h2>
             ©2017-{{ now() | date(format="%Y") }} — <a class="white" href="https://www.vincentprouillet.com">Vincent Prouillet</a> and <a class="white" href="https://github.com/getzola/zola/graphs/contributors">contributors</a>
         </footer>
 
-        <script type="text/javascript" src="{{ get_url(path="elasticlunr.min.js") }}"></script>
-        <script type="text/javascript" src="{{ get_url(path="search.js") }}"></script>
+        <script type="text/javascript" src="{{ get_url(path='elasticlunr.min.js') }}"></script>
+        <script type="text/javascript" src="{{ get_url(path='search.js') }}"></script>
```

Comment:
I prefer `"` to `'`, any reasons to change those?

Reply:
If the editor lsp or formatter is set to regular HTML, it can get hung up on the fact that there are double quotes inside the strings and refuses to work. My reason for this case was that in JS you can use single quotes in double quotes and vice versa, so they should likely be the same in HTML.

I believe the specific reason in this case was that I was using Prettier which was angry that the tags weren't closed due to some weirdness there.

Reply:
good point i didn't think of that